src/main/java/org/elasql/perf/tpart/metric/MetricCollector.java

```diff
@@ -53,8 +59,11 @@ public void run() {
 	private TPartSystemMetrics collectSystemMetrics() {
 		TPartSystemMetrics.Builder builder = new TPartSystemMetrics.Builder(Elasql.serverId());
 
-		// XXX: for demo
-		builder.setFakeMetric(12345);
+		builder.setProcessCpuLoad(bean.getProcessCpuLoad());
+		builder.setSystemCpuLoad(bean.getSystemCpuLoad());
```

Comment:
I saw there is a warning saying that `getSystemCpuLoad()` is deprecated. Is there any other alternative for this?

Reply:
I didn't see the warning message though. I think `getSystemLoadAverage` is the alternative api for this.

Reply:
I found that It is deprecated since Java 14. The alternative is `getCpuLoad` in java 14. 

According to the [official document](https://docs.oracle.com/javase/8/docs/jre/api/management/extension/com/sun/management/OperatingSystemMXBean.html), it is not deprecated.

Reply:
I think that I can accept in either way.